readme.md

```diff
@@ -1,7 +1,7 @@
 # Search bar for Airbnb
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_search-bar-airbnb/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_search-bar-airbnb/report/html_report/)
+- [DEMO LINK](https://Vladyslav-amp.github.io/layout_search-bar-airbnb/)
+- [TEST REPORT LINK](https://Vladyslav-amp.github.io/layout_search-bar-airbnb/report/html_report/)
```

Comment:
Add these links to the PR description
<img width="943" alt="image" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/64750958/fb2c50fb-124d-4fa6-8fcd-c2b9c988c258">